infra_tooling_account/app.py

```diff
@@ -18,12 +18,14 @@
 InfraToolingCommonStack(
     app,
     f"cf-{PROJECT_NAME}-InfraToolingCommonStack-{STAGE_NAME}",
+    tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME},
```

Comment:
Maybe, it's better to define tags var once
tags={"project_name": PROJECT_NAME, "stage_name": STAGE_NAME}
and use it in all stack's params?

Reply:
Done